src/include/wiredtiger.in

```diff
@@ -4999,723 +4999,757 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
-#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_POST_LRU	1114
+#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_POST_LRU	1120
 /*! cache: pages queued for urgent eviction */
-#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_URGENT	1115
+#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_URGENT	1121
 /*! cache: pages queued for urgent eviction during walk */
-#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_OLDEST	1116
+#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_QUEUED_OLDEST	1122
 /*! cache: pages read into cache */
-#define	WT_STAT_CONN_CACHE_READ				1117
+#define	WT_STAT_CONN_CACHE_READ				1123
 /*! cache: pages read into cache after truncate */
-#define	WT_STAT_CONN_CACHE_READ_DELETED			1118
+#define	WT_STAT_CONN_CACHE_READ_DELETED			1124
 /*! cache: pages read into cache after truncate in prepare state */
-#define	WT_STAT_CONN_CACHE_READ_DELETED_PREPARED	1119
+#define	WT_STAT_CONN_CACHE_READ_DELETED_PREPARED	1125
 /*! cache: pages read into cache requiring cache overflow entries */
-#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE		1120
+#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE		1126
 /*! cache: pages read into cache requiring cache overflow for checkpoint */
-#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_CHECKPOINT	1121
+#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_CHECKPOINT	1127
 /*! cache: pages read into cache skipping older cache overflow entries */
-#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_SKIPPED	1122
+#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_SKIPPED	1128
 /*!
  * cache: pages read into cache with skipped cache overflow entries
  * needed later
  */
-#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_DELAY		1123
+#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_DELAY		1129
 /*!
  * cache: pages read into cache with skipped cache overflow entries
  * needed later by checkpoint
  */
-#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_DELAY_CHECKPOINT	1124
+#define	WT_STAT_CONN_CACHE_READ_LOOKASIDE_DELAY_CHECKPOINT	1130
 /*! cache: pages requested from the cache */
-#define	WT_STAT_CONN_CACHE_PAGES_REQUESTED		1125
+#define	WT_STAT_CONN_CACHE_PAGES_REQUESTED		1131
 /*! cache: pages seen by eviction walk */
-#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_SEEN		1126
+#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_SEEN		1132
+/*! cache: pages seen by eviction walk that are already queued */
+#define	WT_STAT_CONN_CACHE_EVICTION_PAGES_ALREADY_QUEUED	1133
 /*! cache: pages selected for eviction unable to be evicted */
-#define	WT_STAT_CONN_CACHE_EVICTION_FAIL		1127
+#define	WT_STAT_CONN_CACHE_EVICTION_FAIL		1134
+/*!
+ * cache: pages selected for eviction unable to be evicted because of
+ * active children on an internal page
+ */
+#define	WT_STAT_CONN_CACHE_EVICTION_FAIL_ACTIVE_CHILDREN_ON_AN_INTERNAL_PAGE	1135
+/*!
+ * cache: pages selected for eviction unable to be evicted because of
+ * failure in reconciliation
+ */
+#define	WT_STAT_CONN_CACHE_EVICTION_FAIL_IN_RECONCILIATION	1136
+/*!
+ * cache: pages selected for eviction unable to be evicted because of
```

Comment:
Could you clarify what overflow keys are?

Reply:
The keys that are more than the configured max size of a page (internal and leaf) are considered as overflow keys. These type of key/values are stored with a different cell type `WT_CELL_KEY_OVFL` and etc. There are some stats around overflow keys: `rec_overflow_key_internal`, `rec_overflow_key_leaf` and `rec_overflow_value`.